src/pages/ProductDetails/product.details.page.tsx

```diff
@@ -1,8 +1,123 @@
+  const dispatch: AppDispatch = useDispatch();
+  const { product } = useAppSelector((state) => state.product);
+
+  useEffect(() => {
+    if (productId) {
+      dispatch(getProduct({ _id: productId }));
+    }
+    return () => {
+      dispatch(clearProduct());
+    };
+  }, []);
+
+  if (!product) {
+    return "Loading...";
+  }
+
   return (
-    <div className="p-4">
-      <h1 className="text-2xl font-bold">Product Details Page (PDP)</h1>
-      <p>Details of the product go here!</p>
+    <div className="grid grid-cols-[1fr,1fr] gap-10">
+      <div>
+        <img src="/frame.svg" alt="" />
+      </div>
+      <div className="flex flex-col gap-12 items-start">
+        <div className="flex flex-col gap-4 h-[50px]">
+          <h1 className="text-3xl font-bold text-black">
+            {product?.title}
+          </h1>
+          <div className="flex item-center gap-4">
+            <ProductRating value={product?.rating ?? 0} />
+            <span className="text-grayText">{`(10 customers reviewed)`}</span>
+          </div>
+        </div>
+        <div>{product?.description}</div>
+
+        {/*Info ection with grid*/}
+        <div className="grid grid-cols-[1fr,1fr] gap-x-16 gap-y-4 w-full">
+          <div className="grid grid-cols-[1fr,2fr] gap-8">
+            <span className="text-grayText">Country</span>
+            <span>{product?.country}</span>
+          </div>
+          <div className="grid grid-cols-[1fr,2fr] gap-8">
+            <span className="text-grayText">Category</span>
+            <span>{product?.country}</span>
+          </div>
+          <div className="grid grid-cols-[1fr,2fr] gap-8">
+            <span className="text-grayText">Brand</span>
+            <span>{product?.brand}</span>
+          </div>
+          {product?.additionalInformation?.map((item) => (
+            <div className="grid grid-cols-[1fr,2fr] gap-8" key={item.key}>
+              <div className="text-grayText">{item.key}</div>
+              <div className="">{item.value}</div>
+            </div>
+          ))}
+        </div>
+        {/*Section with price*/}
+        <div className="h-[89px] border border-grayBorder rounded-2xl flex items-center justify-between px-[24px] w-full">
+          <div>
+            <div className="text-2xl font-bold text-black">{`${product?.price} USD`}</div>
+            <div>800 USD</div>
+          </div>
+          <div className="flex">
+            <Button>+ Add to card</Button>
+          </div>
+        </div>
+
+        {/*Ghost button*/}
+        <Button variant="text">
+          <div className="font-bold flex gap-1 items-center">
+            <img src="/like.svg" />
+            <span>Add to my wish list</span>
+          </div>
+        </Button>
+
+        {/*Description section*/}
+
+        <div className="w-full p-2 flex justify-between border-b border-grayBorder">
+          {["Description", "Reviews", "Questions"].map((tab) => (
```

Comment:
Uneeded comments and empty lines. It will be better to create component if you want to separate it